usage/index.rst

```diff
@@ -38,6 +38,47 @@ Most of the computational power of the system is accessed through the
 batch scheduler, and so demanding applications should be submitted to it
 (see "Running Jobs").
 
+
+.. _usage-mfa:
+
+Multi Factor Authentication
+---------------------------
+
+.. note::
+
+   We are gradually rolling out MFA, so this may not yet apply to your account.
+
+When connecting to the login nodes, the system has to validate your
+identity. In order to help keep accounts secure, Bede employs a
+technique called Multi Factor Authentication, where something you
+know (your password), is combined with something you have (an app on
+a mobile device, such as your phone).
+
+When you first login to Bede, or after a password reset, you will be prompted
+to:
+
+1. Change your password
+2. Import an MFA token into an app on your mobile device
+
+Instead of `Password`, to login after this you will be prompted for:
+
+- `First Factor` - this is your password
```

Comment:
In RST single backticks are "interpreted text", which is generally combined with an extension to have concrete meaning. 

I'm assuming you meant to mark it as inline ala markdown? in which case it's double backticks. Alternatively `*` or `**` for emphasized (italics) and strongly emphasized (bold) text. 

```suggestion
- ``First Factor`` - this is your password
```